plugins/inventory/lxd.py

```diff
@@ -0,0 +1,1107 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Frank Dornheim <dornheim@posteo.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    name: lxd
+    short_description: Returns Ansible inventory from lxd host
+    description:
+        - Get inventory from the lxd
```

Comment:
```suggestion
        - Get inventory from the lxd.
```
Descriptions must be proper sentences. Please also check all other descriptions. (The only exception is `short_description`.)

Reply:
Hi Felix , i will try my best.